deploy/db_setup.py

```diff
@@ -0,0 +1,50 @@
+from pymongo import MongoClient
```

Comment:
That would probably be great to have that code shared with the classic installer script.
I also realize DB indexes are not automatically built, but that's not the matter of this PR.

Reply:
Tracked in https://github.com/UCL-INGI/INGInious/issues/968.